net-misc/gsasl/gsasl-1.8.0.recipe

```diff
@@ -0,0 +1,59 @@
+SUMMARY="The GNU SASL tool"
+DESCRIPTION="GNU SASL is an implementation of the Simple Authentication and \
+Security Layer framework and a few common SASL mechanisms. SASL is used by \
+network servers (e.g., IMAP, SMTP) to request authentication from clients, and \
+in clients to authenticate against servers.
+
+Supported mechanisms are ANONYMOUS, EXTERNAL, LOGIN, PLAIN, SECURID, NTLM, \
+DIGEST-MD5, CRAM-MD5, SCRAM-SHA-1, SCRAM-SHA-1-PLUS, GS2-KRB5, GSSAPI."
+HOMEPAGE="https://www.gnu.org/software/gsasl/"
+COPYRIGHT="2004-2012 Simon Josefsson"
+LICENSE="GNU GPL v3
+	GNU LGPL v3"
+REVISION="1"
+SOURCE_URI="https://ftp.gnu.org/gnu/gsasl/gsasl-$portVersion.tar.gz"
+CHECKSUM_SHA256="310262d1ded082d1ceefc52d6dad265c1decae8d84e12b5947d9b1dd193191e5"
+PATCHES="gsasl-$portVersion.patchset"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+
+PROVIDES="
+	gsasl = $portVersion
+	cmd:gsasl = $portVersion
+	"
+REQUIRES="
+	haiku
+	lib:libgsasl
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	devel:libgsasl
+	"
```

Comment:
```diff
 BUILD_REQUIRES="
 	haiku_devel
+	devel:libgnutls
 	devel:libgsasl
+	devel:libiconv
+	devel:libintl
 	"
 BUILD_PREREQUIRES="
+	cmd:aclocal
+	cmd:autoconf
+	cmd:autoheader
+	cmd:automake
 #	cmd:dia
```
I'm trying to see if we could call `make` without the `-i` and avoid the error with by changing something, maybe this line in `configure.ac`:
```
AM_MISSING_PROG(DIA, dia, $missing_dir)
```
But I did not succeed yet.
`dia` seems to be needed to create a png file. So I guess this is only for the docs.
I tried to pass `--disable-gtk-doc --disable-gtk-doc-html` to configure but it does not help...